v3/docs/gs/sysreqs.md

```diff
@@ -8,18 +8,25 @@ The following are system requirements for running HSE applications.
 Hardware requirements are largely dictated by the application embedding HSE
 and the amount of data stored.  The following are general guidelines.
 
-* **Architecture**: 64-bit Intel&reg; and AMD (x86_64); 64-bit IBM Z&reg; (s390x)
+* **Architecture**: 64-bit Intel&reg; or AMD (x86_64); 64-bit IBM Z&reg; (s390x)
 * **Memory**: 32 GB or more
 * **Block Storage**: SSD volumes *only*; use NVMe for best performance
 * **Persistent Memory** (optional): must support a DAX-enabled file system
 
-If a KVDB [media class](storage.md#media-classes) is configured on multiple
-block storage devices, such
-as when using XFS with LVM, performance can be **significantly** improved by
-balancing these devices across NUMA nodes.
-Tools like [`lstopo`](https://linux.die.net/man/1/lstopo) can
-be helpful in creating and verifying a balanced configuration.
+If a file system storing a KVDB [media class](storage.md#media-classes) is configured on a
+logical block device comprising multiple physical block devices, such as when using XFS with LVM,
+performance can be **significantly** improved by balancing these physical devices across NUMA nodes.
+Tools like [`lstopo`](https://linux.die.net/man/1/lstopo) can be helpful in creating and
+verifying a balanced configuration.
 
+For the single logical or physical block device configured with a file system for use with HSE,
+performance can also be **significantly** improved by setting the read-ahead to the greater
```

Comment:
Should you mention which setting? And also say it is experimental?

Reply:
I am not aware that read_ahead_kb is experimental.  It's been in the distro docs for a long time.

Reply:
I was referring to the HSE rparam, but I see that you are referring to changing the system parameter. Nevermind this comment.